_posts/2019-06-18-list-strategy.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: posts
+title: "Automatic execution strategy selection in Bazel 0.27"
+authors:
+  - ishikhman
+---
+**tl;dr**: Exciting news! After 0.27 release Bazel will *auto select* a suitable execution strategy if none is provided. Customization and strategy enforcement will still be possible and will be much easier to configure.
+
+## Before Bazel 0.27
+
+It was possible to configure Bazel's execution strategy for a build via flags `--spawn_strategy=`, `--strategy=Mnemonic=` and `--strategy_regexp=Regex=`. This mechanism was quite powerful and widely used, but had some drawbacks:
+
+- Bazel's defaults did not account for remote execution at all, therefore we had to make sure to set the execution strategy for every mnemonic manually.
+
+- Starlark rules providing a persistent worker had to ship a .bazelrc file that sets --strategy=Mnemonic=worker (i.e. look at rules_scala). 
+
+- Bazel has hardcoded defaults for native action mnemonics. For example, Bazel will use sandboxing (if it's available) for all actions by default but has a hardcoded default to use persistent workers for Java compilation actions.
+
+- To configure fallback strategy one had to provide additional flag `--remote_local_fallback_strategy=`.
+
+All of the above are now unnecessary with this change and the default behaviour is much more reliable and predictable.
```

Comment:
_this change_: what change? You haven't described it yet. Maybe "starting from Bazel 0.27" or "with automatic strategy selection". Or maybe just drop this entire statement.

Reply:
dropped